src/Microsoft.AspNet.OData.Shared/Formatter/EdmLibHelpers.cs

```diff
@@ -161,6 +164,18 @@ private static IEdmType GetEdmType(IEdmModel edmModel, Type clrType, bool testCo
                     clrType = underlyingType;
                 }
 
+                if (_cache.TryGetValue(edmModel, out var cachedClrTypes))
+                {
+                    if (cachedClrTypes.TryGetValue(clrType, out var cachedType))
```

Comment:
Please use IEdmType instead of var. We usually try to avoid using var in our libs

Reply:
will replace all var with solid types.

Reply:
Done.